examples/foc/Kconfig

```diff
@@ -98,6 +98,26 @@ config EXAMPLES_FOC_SENSORED
 
 endchoice #
 
+menu "Motor phy"
+
+config EXAMPLES_FOC_MOTOR_POLES
+	int "FOC example motor poles pairs"
+	default 0
+
+config EXAMPLES_FOC_MOTOR_RES
+	int "FOC example motor phase resistance (x1000000)"
+	default 0
+
+config EXAMPLES_FOC_MOTOR_IND
+	int "FOC example motor phase inductance (x1000000)"
+	default 0
+
+config EXAMPLES_FOC_MOTOR_FLUXLINK
+	int "FOC example motor flux link (x1000000)"
+	default 0
```

Comment:
Please include a help explaining about the value for each unit. The (x1000000) was not simple to figure-out until looking the source code. It could be a "doesn't care" bit or even a hex code.

Reply:
done